web-viewer/src/components/Map.jsx

```diff
@@ -14,7 +13,6 @@ const Map = React.createClass({
       layers: PropTypes.arrayOf(CustomPropTypes.baseLayer),
       active: PropTypes.string
     }),
-    activeWeatherLayerId: PropTypes.string
   },
   getInitialState() {
     this.layerCache = new LayerCache()
```

Comment:
I think semantically that this should go in `componentDidMount` or `initializeLayerCache`